src/forms/Button.jsx

```diff
@@ -69,25 +71,38 @@ class Button extends React.PureComponent {
 			</Flex>
 		);
 
+		const ButtonTag = buttonTag;
+		const hrefAttr = buttonTag === 'Link' && buttonTag !== 'button' ? 'to' : 'href';
+		const buttonProps = {
+			className: classNames.button,
+			onClick: onClick,
+			role: buttonTag ? 'button' : null,
+			[hrefAttr]: linkTo,
+			...other
+		};
+
 		return (
-			<button
-				className={classNames.button}
-				onClick={onClick}
-				role='button'
-				{...other}
+			<ButtonTag
+				{...buttonProps}
 			>
 				{ icon ? iconChildren : children }
-			</button>
+			</ButtonTag>
 		);
 	}
 }
 
+Button.defaultProps = {
+	buttonTag: 'button'
+};
+
 Button.propTypes = {
 	reset: PropTypes.bool,
 	fullWidth: PropTypes.bool,
 	primary: PropTypes.bool,
 	small: PropTypes.bool,
 	icon: PropTypes.any,
 	right: PropTypes.bool,
+	buttonTag: PropTypes.oneOf(['button', 'a', 'Link']),
+	linkTo: PropTypes.string
```

Comment:
we should probably do a custom prop validator here to make `linkTo` required if `buttonTag` is 'Link'